test/src/cookies.spec.ts

```diff
@@ -27,159 +27,190 @@ describe('Cookie specs', () => {
 
   describe('Page.cookies', function () {
     it('should return no cookies in pristine browser context', async () => {
-      const {page, server} = await getTestState();
-      await page.goto(server.EMPTY_PAGE);
-      await expectCookieEquals(await page.cookies(), []);
+      const {page, server, close} = await launch({});
```

Comment:
Temporary switch to re-launching browser, as incognito mode is not implemented in BiDi